src/collective/ckeditor/browser/statics/ckeditor_plone.js

```diff
@@ -64,29 +66,43 @@ if ( window.CKEDITOR )
 
 			html += '</p><p>' + ck_mf(locale.ckeditorStillUsable) + '</p>';
 
-			document.getElementById( 'alerts' ).innerHTML = html;
+			let alerts = document.getElementById( 'alerts' );
+            if (alerts) {
+                alerts.innerHTML = html;
+            }
 		};
 
+        var init = function(i18n){
+            let lang = $('html').attr('lang');
+            // get the translation tool catalog for the given language and domain
+            i18n.loadCatalog('collective.ckeditor', lang);
+            // initialize the message factory
+            ck_mf = i18n.MessageFactory('collective.ckeditor', lang);
+        }
+
 		$(document).ready(function() {
-            /* Setting up jsi18n message factory for ckeditor with jsi18n (from mockup) */
-            require(['mockup-i18n'], function(I18N) {
-                try {
+            /* Setting up jsi18n message factory for ckeditor with jsi18n (from mockup) via requirejs.
+                Use a fallback for plone 4 without require but
+            */
+            try {
+                require(['mockup-i18n'], function(I18N) {
                     if (typeof(i18n) === 'undefined') {
-                        var i18n = new I18N();
+                        i18n = new I18N()
                     }
-                    var lang = $('html').attr('lang');
-                    // get the translation tool catalog for the given language and domain
-                    i18n.loadCatalog('collective.ckeditor', lang);
-                    // let's initialize message factory
-                    ck_mf = i18n.MessageFactory('collective.ckeditor', lang);
-                } catch (e) {
+                    init(i18n);
+                });
+            }
+            catch(e) {
+                try {
+                    init(jarn.i18n);
+                } catch(e) {
                     console.log('failed to load i18n');
```

Comment:
Some sort of context information whould be nice, which hints the developper that it happens in collective.ckeditor.

Reply:
Done.